sdk/armada-protocol-common/src/common/interfaces/IRebalanceData.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  AddressDataSchema,
+  IAddress,
+  ITokenAmount,
+  TokenAmountDataSchema,
+} from '@summerfi/sdk-common/common'
+import { z } from 'zod'
+
+/**
+ * Unique signature to provide branded types to the interface
+ */
+export const __signature__: unique symbol = Symbol()
+
+/**
+ * @name IRebalanceData
+ * @description Data structure for rebalancing assets, used by Keepers of a fleet
+ */
+export interface IRebalanceData {
```

Comment:
just heads up that it will be changed after latest armada PRs are merged



```ts
struct RebalanceData {
    address fromArk;
    address toArk;
    uint256 amount;
    bytes boardData;
    bytes disembarkData;
}
```

Reply:
Yeah, this is probably on-going work. We need to sync constantly about this to make sure we don't miss any changes. For now I'll base this PR in the current changes on `main`